### DISCUSSION TITLE
VS Code 1.92.2 repeatedly downloads the VS Code Server and installs failed.

### DISCUSSION THREAD

**Discussion by pingfandeshijie9394** [coder/code-server#6981] [Q&A]
VS Code version
1.92.2
fee1edb8d6d72a0ddff41e5f71a671c23ed924b9
x64

local machine：Windows 10 Enterprise LTSC (Long-Term Servicing Channel). X86
remote server：Linux 5.15.0-113-generic #123~20.04.1-Ubuntu

questions：
localMachine and remote server has the same vs code commit-id
use remote-ssh to connect to remote server is repeated downloading and installing vs code server
with errmsg is
[server] Error installing server: error checking server integrity: failed to run command "/home/xxxxx/.vscode-server/cli/servers/Stable-fee1edb8d6d72a0ddff41e5f71a671c23ed924b9.staging/server/bin/code-server" (code -1): Permission denied (os error 13)

tried manual install commit-id vs code server is not work. 

how to fix this questions?

**Comment by code-asher** [comment]
Looks like this is about local VS Code with the Remote SSH plugin, not code-server (a web wrapper for VS Code)!  Please send your question to https://github.com/microsoft/vscode

**Reply by pingfandeshijie9394** [reply]
OK, thanks
